docs/src/main/sphinx/release/release-354.md

```diff
@@ -0,0 +1,56 @@
+# Release 354 (XXXXXXXX Mar 2021)
```

Comment:
March

Reply:
We use 3-letter abbreviations for months: https://trino.io/docs/current/release.html